[Yoshino] Fingerprint build broken

When building yoshino, build runs into following error:
```
[ 40% 49512/121165] target  C++: android.hardware.biometrics.fingerprint@2.1-service.sony <= vendor/oss/fingerprint/BiometricsFingerprint.cpp
FAILED: out/target/product/lilac/obj/EXECUTABLES/android.hardware.biometrics.fingerprint@2.1-service.sony_intermediates/BiometricsFingerprint.o
/bin/bash -c "PWD=/proc/self/cwd /usr/bin/ccache prebuilts/clang/host/linux-x86/clang-r353983c/bin/clang++      -I vendor/oss/fingerprint -I out/target/product/lilac/obj/EXECUTABLES/android.hardware.biometrics.fingerprint@2.1-service.sony_intermediates -I out/target/product/lilac/gen/EXECUTABLES/android.hardware.biometrics.fingerprint@2.1-service.sony_intermediates \$(cat out/target/product/lilac/obj/EXECUTABLES/android.hardware.biometrics.fingerprint@2.1-service.sony_intermediates/import_includes)   -isystem out/target/product/lilac/obj/include -isystem kernel/sony/msm-4.9/common-headers/kernel-headers -c  -Werror=implicit-function-declaration -DANDROID -fmessage-length=0 -W -Wall -Wno-unused -Winit-self -Wpointer-arith -no-canonical-prefixes -DNDEBUG -UDEBUG -fno-exceptions -Wno-multichar -O2 -g -fno-strict-aliasing -fdebug-prefix-map=/proc/self/cwd= -D__compiler_offsetof=__builtin_offsetof -faddrsig -Wimplicit-fallthrough -Werror=int-conversion -Wno-reserved-id-macro -Wno-format-pedantic -Wno-unused-command-line-argument -fcolor-diagnostics -Wno-zero-as-null-pointer-constant -Wno-sign-compare -Wno-defaulted-function-deleted -Wno-inconsistent-missing-override -ffunction-sections -fdata-sections -fno-short-enums -funwind-tables -fstack-protector-strong -Wa,--noexecstack -D_FORTIFY_SOURCE=2 -Wstrict-aliasing=2 -Werror=return-type -Werror=non-virtual-dtor -Werror=address -Werror=sequence-point -Werror=date-time -Werror=format-security -nostdlibinc -march=armv8-a -mcpu=cortex-a53 -target aarch64-linux-android -Bprebuilts/gcc/linux-x86/aarch64/aarch64-linux-android-4.9/aarch64-linux-android/bin  -Wsign-promo -D_LIBCPP_ENABLE_THREAD_SAFETY_ANNOTATIONS -Wno-thread-safety-negative -Wno-gnu-include-next -fvisibility-inlines-hidden  -std=gnu++17   -fno-rtti -Wall -DUSE_FPC_YOSHINO -DPLATFORM_SDK_VERSION=29 -fexceptions -std=c++1z -D__ANDROID_API__=29 -D__ANDROID_VNDK__ -fPIE -D_USING_LIBCXX -DANDROID_STRICT   -Werror=int-to-pointer-cast -Werror=pointer-to-int-cast -Werror=address-of-temporary -Werror=return-type -Wno-tautological-constant-compare -Wno-tautological-type-limit-compare -Wno-tautological-unsigned-enum-zero-compare -Wno-tautological-unsigned-zero-compare -Wno-c++98-compat-extra-semi -Wno-return-std-move-in-c++11  -MD -MF out/target/product/lilac/obj/EXECUTABLES/android.hardware.biometrics.fingerprint@2.1-service.sony_intermediates/BiometricsFingerprint.d -o out/target/product/lilac/obj/EXECUTABLES/android.hardware.biometrics.fingerprint@2.1-service.sony_intermediates/BiometricsFingerprint.o vendor/oss/fingerprint/BiometricsFingerprint.cpp"
vendor/oss/fingerprint/BiometricsFingerprint.cpp:534:57: error: too many arguments to function call, expected single argument 'gid', have 2 arguments
                int grp_err = __setActiveGroup(mDevice, gid);
                              ~~~~~~~~~~~~~~~~          ^~~
vendor/oss/fingerprint/BiometricsFingerprint.cpp:216:1: note: '__setActiveGroup' declared here
int BiometricsFingerprint::__setActiveGroup(uint32_t gid) {
^
1 error generated.
09:06:11 ninja failed with: exit status 1
```
Removing the mDevice parameter should fix the build error. Problem is caused by https://github.com/sonyxperiadev/vendor-sony-oss-fingerprint/commit/75d6aa806f1ebf33252a87f30060411c8a1360b1

--- a/BiometricsFingerprint.cpp
+++ b/BiometricsFingerprint.cpp
@@ -531,7 +531,7 @@ void BiometricsFingerprint::AuthenticateAsync() {
                 result = fpc_init(&mDevice->fpc, mWt.getEventFd());
                 LOG_ALWAYS_FATAL_IF(result < 0, "REINITIALIZE: Failed to init fpc: %d", result);
 #ifdef USE_FPC_YOSHINO
-                int grp_err = __setActiveGroup(mDevice, gid);
+                int grp_err = __setActiveGroup(gid);
                 if (grp_err)
                     ALOGE("%s : Cannot reinitialize database", __func__);
 #else

